src/components/Burst.js

```diff
@@ -32,9 +38,47 @@ class Burst extends Component {
     this.setState(newState);
   }
 
-  handleRequestBurst() {
-    const { requestBurst, expanded, ...imgOpts } = this.props;
-    requestBurst({ ...this.state, ...imgOpts });
+  async handleRequestBurst() {
+    this.setState({ isCapturing: true });
+    const { requestBurst, expanded, frames, frameIDs, ...imgOpts } = this.props;
+    // grab calculator state prior to capture
```

Comment:
With some of your nicely named function names, some of these comments may not be necessary

Reply:
Done